File: pyjl/tests/dataclass_test.py
from dataclasses import dataclass

@dataclass(init=True, repr=True, eq=True, order=True, unsafe_hash=False, frozen=False)
class ValueHolder:
    val: int
    strVal: str

if __name__ == "__main__":
    a = ValueHolder(10, "1")
    print(a)
    # ValueHolder.__init__(a, 2, "10")
    # print(a)

File: pyjl/tests/dataclass_test.jl
using Dataclass

@dataclass mutable struct ValueHolder
val::Int64
strVal::String
_initvars = [_init=true, _repr=true, _eq=true, _order=true, _unsafe_hash=false, _frozen=false]
end


function main()
a = ValueHolder(10, "1")
println(a);
__init__(a, 2, "10");
println(a);
end

main()
